docs/team/member-guide.md

```diff
@@ -0,0 +1,96 @@
+["how can I contribute"](https://jupyterhub-team-compass.readthedocs.io/en/latest/contributing.html)
+guide in the team-compass.
+   
+Don't forget that, as a member of the team, you're representing the community
+when you interact with people online. Try to keep a friendly, positive
+attitude, and be welcoming and helpful in bringing others into the community
+and answering their questions.
+   
+### Are there any specific responsibilies?
+   
+We don't want team membership to
+be a big burden (many of us have one or more other jobs too!) but there are
+a few things that you should do as a new team member:
+   
+1. **"Watch" the [team compass repository](https://github.com/jupyterhub/team-compass)**
+   so that you're notified when team conversations are happening.
+2. **"Watch" the [Discourse jupyterhub team category](https://discourse.jupyter.org/c/jupyterhub/internal)**
+   for the same reason.
+3. **Semi-regularly attend team meetings**. You can find a calendar of upcoming
+   meetings [on the team meetings page](https://jupyterhub-team-compass.readthedocs.io/en/latest/meetings.html).
+4. **Follow the [Binder team responsibilities](https://jupyterhub-team-compass.readthedocs.io/en/latest/binder/governance.html#team-responsibilities) guidelines**.
+   While this is scoped for the Binder team, it is a good rubric to follow for
+   JupyterHub as well.
+   
+## When should I merge a pull request?
+
+As a team member, you're encouraged to help others contribute to the project
+by reviewing their code, guiding them towards making a contribution and
+improving it, and ultimately merging their contribution into the project.
+
+Having merge rights is both a privilege and a responsibility - please be
+thoughtful when using it! To that extent, here are a few guidelines when
+deciding to merge things into one of our repositories:
+
+* **Make sure it's quality code**. We know this is somewhat subjective, but
+  ensure that the code is well-organized and thoughtfully-written, that any
+  new features are documented, and that it abides by best-practices in Python,
+  JavaScript, etc.
+* **Make sure there are tests**. We try not to merge any new features (or
+  bugfixes!) without adding tests for them. It's easy to consider something
+  minor-enough that it doesn't warrant a test, but try to avoid doing this!
+  Adding tests usually only takes a moment, and our future selves will thank
+  us for it later.
+* **Make sure there's been enough time for discussion**. We're an open
+  community with an inclusive decision-making process. This means that
+  sometimes we need to slow down to make sure others have a chance to
+  review and provide their thoughts on changes. There's no hard rule for
+  this, but try to make sure people have a chance to weigh in. Consider
+  pinging people that you think might be interested in a question, and
+  give it a few extra days before merging if you think a topic will be
+  complex enough to warrant discussion.
```

Comment:
Maybe we can add a bullet point that encourages people to "just merge it". It is a little in contrast with the "leave time for discussion" point. I think making it clear that not everything needs a long time for discussions nor that everyone will look at every PR and weigh in on it.

Something to convey: "We each trust our team members, this is why we are a team. If you think something is good to go, go for it. Most of the time you will be right and by taking action you will move the project forwards instead of stalling. Sometimes it will turn out that we need to keep iterating or even undo an action. This is fine. When this does happen please be considerate, both as the one who took action and the one who opposes it. It is a sign that there is conflict and situations with conflict have a habit of deteriorating unless everyone makes an extra effort. Luckily there are very few actions in our projects that can't be edited and improved on."

Reply:
This section is about merging code but while thinking about my last comment I realised that actually this applies to all things we do in the name of the team. Posting on twitter, discourse, giving talks, commenting on Reddit, merging PRs, deploying changes to mybinder.org, etc.

My rule of thumb is: would someone else be surprised if they see this action? If yes then we need to talk (more) about it, if no do it now.

The thinking is that most people don't like things that take them by surprise. So much so that they will be against ideas that they like just because they were surprised by it. For example: airlines that slap a "baggage fee" on to the ticket price right at the very last moment after not making an effort to put that idea in your head from the start. I don't know anyone who has a good feeling when they discover at the last moment that their ticket will be $300 plus $30 for bags. Where as they'd probably pay $335 for the same ticket if from the beginning it was clear that this is the "all in, no sneaky last minute fees" price, even though it is higher.

Reply:
yeah - I think that a reasonable way to summarize this topic is "as a member of the JupyterHub teams, we trust your judgment, and we ask you to use your best judgment in deciding when to take an action". Will add in that language. I can also add in a suggestion about erring on the side of action instead of inaction.

Reply:
> add in a suggestion about erring on the side of action instead of inaction.

yes please. I often find myself wondering "I could merge this now, but maybe not, maybe I should play it safe and ask someone else for input" and then the PR drops off the front page my notifications and is never acted upon again. -> If I am unsure about "just merging it" then we'd help at least one person by having a sentence about action not inaction in the guide :)